packages/ui-a11y-content/src/AccessibleContent/index.tsx

```diff
@@ -55,7 +55,8 @@ class AccessibleContent extends Component<AccessibleContentProps> {
 
   render() {
     const { alt, children, ...props } = this.props
-    const ElementType = getElementType(AccessibleContent, this.props)
+    // any cast is needed to prevent Expression produces a union type that is too complex to represent errors
+    const ElementType = getElementType(AccessibleContent, this.props) as any
```

Comment:
i have not idea why these errors kept popping up, when I added a `@ts-expect-error` another popped up. And I did not upgrade TypeScript... I guess these super complex props really push TS to the limits.

Reply:
Yeah, I encountered the same thing the other day, somehow we have to fix this type.